src/pages/AdminPage/AdminPage.tsx

```diff
@@ -400,6 +440,22 @@ const AdminPage = () => {
         perPage={15}
         filter={filterUser}
         sortable
+        placeholder={
```

Comment:
this will never appear, because the admin themselves is in the circle already

Reply:
I noticed :/